app/src/main/java/com/example/seminar1/fragment/CameraFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.seminar1.fragment
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.seminar1.databinding.FragmentCameraBinding
+
+class CameraFragment : Fragment() {
+    private var _binding : FragmentCameraBinding? = null
+    private val binding get() = _binding!!
```

Comment:
이제 슬슬 이 보일러 플레이트에 익숙해졌다면 코틀린의 위임 속성을 이용하여 보일러 플레이트를 제거하는 것도 좋습니다.

[AutoClearedValue.kt](https://github.com/SOPT-30-ANDROID-1/Harlequin/blob/main/core/src/main/java/happy/mjstudio/core/presentation/util/AuthClearedValue.kt)

```suggestion
    private var binding: FragmentCameraBinding by AutoClearedValue()
```

한 줄로만 쓸 수 있고 `onDestoryView`를 오버라이딩 하지 않아도 됩니다.